pkg/image/apiserver/apiserver.go

```diff
@@ -119,11 +125,36 @@ func (c *completedConfig) V1RESTStorage() (map[string]rest.Storage, error) {
 }
 
 func (c *completedConfig) newV1RESTStorage() (map[string]rest.Storage, error) {
-	// TODO: allow the system CAs and the local CAs to be joined together.
-	importTransport, err := restclient.TransportFor(&restclient.Config{})
+	cfg := restclient.Config{}
+
+	tlsConfig := &tls.Config{}
+
+	var err error
+	tlsConfig.RootCAs, err = x509.SystemCertPool()
+	if err != nil {
+		return nil, fmt.Errorf("unable to get system cert pool for default transport for image importing: %v", err)
+	}
+	if tlsConfig.RootCAs == nil {
+		tlsConfig.RootCAs = x509.NewCertPool()
+	}
+
+	if _, err := os.Stat("/etc/origin/master/imageimportcerts/cert.pem"); !os.IsNotExist(err) {
```

Comment:
i think this will have to be plumbed from master config, right @deads2k ?

Reply:
> i think this will have to be plumbed from master config, right @deads2k ?

Yeah.  I suspect WIP really means it here and he didn't mean to cause us heart failure. :)

Reply:
it really doesn't.  /etc/origin/master is always mounted already, this is taking advantage of that to say "if you want to provide additional inputs, drop them in this dir".

Reply:
i've updated this to get a path from the master-config instead of hardcoding one.

There is also no default, so you have to set it explicitly in master-config and then ensure the file exists/is mounted into the apiserver pod.